src/main/java/io/jenkins/plugins/luxair/ImageTag.java

```diff
@@ -1,32 +1,32 @@
 package io.jenkins.plugins.luxair;
 
+import kong.unirest.*;
+import kong.unirest.json.JSONObject;
+
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 import java.util.logging.Logger;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
-import kong.unirest.GetRequest;
-import kong.unirest.HttpResponse;
-import kong.unirest.Interceptor;
-import kong.unirest.JsonNode;
-import kong.unirest.Unirest;
-import kong.unirest.json.JSONObject;
 
 
 public class ImageTag {
 
     private static final Logger logger = Logger.getLogger(ImageTag.class.getName());
     private static final Interceptor errorInterceptor = new ErrorInterceptor();
 
+    private ImageTag() {
+        throw new IllegalStateException("Utility class");
+    }
+
     public static List<String> getTags(String image, String registry, String filter, String user, String password) {
 
         String[] authService = getAuthService(registry);
         String token = getAuthToken(authService, image, user, password);
         List<String> tags = getImageTagsFromRegistry(image, registry, token);
         return tags.stream().filter(tag -> tag.matches(filter))
-            .map(tag -> image + ":" + tag)
+            .sorted()
```

Comment:
Please do not sort. I want to have the last pushed tag in first in the selection box, or better sorted using maven versioning (e.g. v2.1.2, v2.1.0, v1.12.4, v1.7.23, v0.2.4)

Reply:
Hmmm okay, sure can do. 
Would it be okay with you if the sort order where configurable? 
Or should I open another PR for something like that?

Reply:
Let split the feature (configurable order) in a other PR.

Reply:
due to the nature of tags allowing free form strings (e.g.: latest or lts) make it not really straight forward two compare them properly (even things like v1.0-alpine would be problematic).